IFC4x3/Properties/d/DynamicRange_1_5uyUpJHENuvUg5DlNq32/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the dynamic range of the earthquake sensor or accelerometer.
```

Comment:
Could be generalized.
```suggestion
Indicates the dynamic range of the sensor.
```

Reply:
Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/1e8b0740cc2e7e000d58ef2f240b7e5a49640ee9